web/annotation_editor_layer_builder.css

```diff
@@ -188,6 +188,101 @@
       border: var(--focus-outline-around);
     }
   }
+
+  .editToolbar {
+    --editor-toolbar-delete-image: url(images/editor-toolbar-delete.svg);
+    --editor-toolbar-bg-color: #f0f0f4;
+    --editor-toolbar-fg-color: #2e2e56;
+    --editor-toolbar-border-color: #8f8f9d;
+    --editor-toolbar-hover-bg-color: #e0e0e6;
+    --editor-toolbar-active-bg-color: #cfcfd8;
+    --editor-toolbar-focus-outline-color: #0060df;
+    --editor-toolbar-shadow: 0 2px 6px 0 rgb(58 57 68 / 0.2);
+
+    @media (prefers-color-scheme: dark) {
+      --editor-toolbar-bg-color: #2b2a33;
+      --editor-toolbar-fg-color: #fbfbfe;
+      --editor-toolbar-border-color: #2b2a33;
+      --editor-toolbar-hover-bg-color: #52525e;
+      --editor-toolbar-active-bg-color: #5b5b66;
+      --editor-toolbar-focus-outline-color: #0df;
+    }
+
+    @media screen and (forced-colors: active) {
+      --editor-toolbar-bg-color: ButtonFace;
+      --editor-toolbar-fg-color: ButtonText;
+      --editor-toolbar-border-color: ButtonText;
+      --editor-toolbar-hover-bg-color: AccentColor;
+      --editor-toolbar-active-bg-color: ButtonFace;
+      --editor-toolbar-focus-outline-color: ButtonBorder;
+      --editor-toolbar-shadow: none;
+    }
+
+    display: flex;
+    width: fit-content;
+    height: 28px;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    cursor: default;
+
+    position: absolute;
+    inset-inline-end: 0;
+    inset-block-start: calc(100% + 6px);
+
+    border-radius: 4px;
+    background-color: var(--editor-toolbar-bg-color);
+    border: 1px solid var(--editor-toolbar-border-color);
+    box-shadow: var(--editor-toolbar-shadow);
+
+    &.hidden {
+      display: none;
+    }
+
+    .icons {
+      display: flex;
+      padding: 0 2px;
+      justify-content: center;
+      align-items: center;
+      gap: 4px;
+
+      .delete {
+        width: 24px;
+        height: 24px;
+        cursor: pointer;
+        border: none;
+        background-color: transparent;
+
+        &::before {
+          content: "";
+          mask-image: var(--editor-toolbar-delete-image);
+          mask-repeat: no-repeat;
+          mask-position: center;
+          display: inline-block;
+          background-color: var(--editor-toolbar-fg-color);
+          width: 100%;
+          height: 100%;
+        }
+      }
+
+      :is(.delete) {
+        &:hover {
+          border-radius: 2px;
+          background-color: var(--editor-toolbar-hover-bg-color);
+        }
+
+        &:active {
+          border-radius: 2px;
+          background-color: var(--editor-toolbar-active-bg-color);
+        }
+
+        &:focus-visible {
+          border-radius: 3px;
+          outline: 2px solid var(--editor-toolbar-focus-outline-color);
+        }
+      }
```

Comment:
Can't this block be combined with the one above?

Reply:
My idea was to prepare the future, but let me slightly change that stuff.